RobotArdu/src/main/java/de/fhg/iais/roberta/worker/compile/ArduinoCompilerWorker.java

```diff
@@ -122,7 +146,17 @@ public void execute(Project project) {
                 project.getRobot(),
                 arduinoArch
             };
-        Pair<Boolean, String> result = Util.runCrossCompiler(executableWithParameters, crosscompilerSource, project.isNativeEditorCode());
+
+        //Pair<Boolean, String> result = Util.runCrossCompiler(executableWithParameters, crosscompilerSource);
+        //System.out.println("************** Script:" + scriptName);
+
+        Pair<Boolean, String> result;
+        if ( SystemUtils.IS_OS_WINDOWS ) {
```

Comment:
in case you follow my comments above, `executableWithParameters` has already the correct parameters and a query is here not needed anymore